### PR TITLE
BLAN-5252 New Loading Spinner: Fix loading circle width size

### PR DESCRIPTION
**Pull request by Thearith**
Just to provide a setter for setting donut radius. We need to be able to set this param in run-time.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,2 +1,7 @@
 # suppress inspection "UnusedProperty" for whole file
-org.gradle.daemon=true
\ No newline at end of file
+org.gradle.daemon=true
+ANDROID_GRADLE_PLUGIN_VERSION=3.3.2
+ANDROID_BUILD_TOOLS_VERSION=28.0.3
+MIN_ANDROID_SDK_VERSION=21
+ANDROID_SDK_VERSION=28
+ANDROID_SUPPORT_LIBRARY_VERSION=28.0.0
```
Comment (gildor) [inline gradle.properties]: Can we easily migrate on AndroidX?

**Review by gildor — CHANGES_REQUESTED** [review]

File: gradle.properties (hunk 1 above)
Comment (Thearith) [reply gradle.properties]: Okay sure.

File: build.gradle [hunk 2]
```diff
@@ -37,5 +39,5 @@ android {
 }
 
 dependencies {
-    compile "com.android.support:support-annotations:24.0.0"
+    implementation "androidx.appcompat:appcompat:$ANDROIDX_VERSION"
```
Comment (gildor) [inline build.gradle]: But looks that we don't need appcompat. `androidx.annotation:annotation:$ANDROIDX_VERSION` should be enough

**Review by gildor — CHANGES_REQUESTED** [review]

**Review by sarah-nkl — APPROVED** [review]

**Comment by Thearith** [comment]
retest this please

**Review by gildor — APPROVED** [review]

**Pull request merged**